### ISSUE TITLE
[iOS] freezes, [1.1.5] fixes it, but don't need Android X support

### ISSUE THREAD

**Issue by jeffmon** [react-native-kit/react-native-track-player#642]
# Configuration

```
 React Native Environment Info:
    System:
      OS: macOS 10.14.4
      CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
      Memory: 426.46 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 12.0.0 - ~/.nvm/versions/node/v12.0.0/bin/node
      Yarn: 1.13.0 - /usr/local/bin/yarn
      npm: 6.9.0 - ~/.nvm/versions/node/v12.0.0/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.2, macOS 10.14, tvOS 12.2, watchOS 5.2
      Android SDK:
        API Levels: 23, 25, 26, 27, 28
        Build Tools: 23.0.1, 25.0.3, 27.0.3, 28.0.1, 28.0.3
        System Images: android-17 | Google APIs Intel x86 Atom, android-23 | Intel x86 Atom_64, android-23 | Google APIs Intel x86 Atom_64, android-24 | Google APIs Intel x86 Atom, android-27 | Google Play Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5056338
      Xcode: 10.2.1/10E1001 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.8.6 => 16.8.6
      react-native: 0.59.8 => 0.59.8
```
`react-native-track-player 1.1.4`

# Issue

App freezes when switching to a new track on iOS, exact same issue here: https://github.com/react-native-kit/react-native-track-player/issues/548. New version `1.1.5` fixes that freezing but now Android isn't working due to Android X support. We don't plan to upgrade `react-native` to `0.60.0` anytime soon due to that version being only a release candidate. Would you be able to have a separate release _without_ Android X support?

**Comment by Guichaguri** [comment]
You can use [jetifier](https://github.com/mikehardy/jetifier) if you don't want to update to 0.60 just yet.

**Comment by jeffmon** [comment]
@Guichaguri I ran `npx jetify -r` but I get a different error after running `react-native run-android`: 

```
> Task :react-native-track-player:compileDebugJavaWithJavac FAILED
/***/node_modules/react-native-track-player/android/src/main/java/com/guichaguri/trackplayer/module/MusicModule.java:12: error: package android.support.v4.content does not exist
import android.support.v4.content.LocalBroadcastManager;
```

**Comment by Guichaguri** [comment]
You should open an issue in [jetifier](https://github.com/mikehardy/jetifier/issues) for that. You can also try going into AndroidX instead of downgrading to the Support Library, that might work better.

**Comment by dcvz** [comment]
I've published [v1.1.6](https://github.com/react-native-kit/react-native-track-player/releases/tag/v1.1.6) which has all the latest changes without Android X support. We'll probably only do this once - while `0.60` is not officially released.
